modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,70 +4,114 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager publishes APIs from your organization's instance of Exchange so they can be accessed by your organization's developer community.
+An ACM community administrator imports APIs, new API versions, and minor API updates from their Exchange organization to their community, reviews them, and publishes them.
 
-== List your community's APIs
+== Import APIs
 
-Open the *Manage API Publication*. This tab displays lists of *APIs in this Community* and *APIs in Exchange*.
+Open the ACM administrator interface and select the *Manage APIs* tab at the top left.
 
-Each of your Anypoint organization's APIs are shown in one of the lists.
+On this tab you can manage what APIs from Exchange are presented in your community and their metadata and visibility.
 
-Each item in the list *APIs in this Community* shows an icon, name, visibility icon, description, and an *Edit* button.
+The section *APIs in this Community* shows the API versions that have been added to the community.
```

Comment:
I'm not sure how you are handling capitalization of community. Sometimes you cap sometimes not. Not sure if Jesus has an opinion on that. But pick one and be consistent.

Reply:
Oops! Should be lowercase throughout, except when quoting UI text where it's capitalized. Fixed.